src/ui/components/Header/UserOptions.js

```diff
@@ -60,19 +60,7 @@ function UserOptions({ recordingId, setModal }) {
         setExpanded={setExpanded}
         expanded={expanded}
         orientation="bottom"
-      >
-        {recordingId ? (
-          <button className="row" onClick={onLibraryClick}>
-            <MaterialIcon>home</MaterialIcon>
-            <span>Library</span>
-          </button>
-        ) : null}
-        {showShare && (
```

Comment:
nice!

Reply:
Thanks! But darn, now we have a conflict. Will check later but feel free to tidy it up if you get a chance. (I assume this collided with the editable title item)